packages/terra-notification-dialog/src/NotificationDialog.module.scss

```
@@ -42,98 +33,120 @@
-    }
+  .header {
+    flex: 0 0 auto;
+    padding: var(--terra-notification-dialog-header-padding, 0 1.42857rem); // left and right padding
+    z-index: 1;
+  }
+  
+  .header-content {
+    border-bottom: var(--terra-notification-dialog-header-content-border-bottom, 1px solid #dedfe0);
+    display: flex;
+    flex-direction: row;
+    padding: var(--terra-notification-dialog-header-content-padding, 1.35714rem 0 0.5rem); // top and bottom padding to squish border bottom
+    text-align: left;
+    width: 100%;
+  }
 
-    > .success {
-      background: var(--terra-notification-dialog-success-icon-background, inline-svg("<svg viewBox='0 0 49 49' xmlns='http://www.w3.org/2000/svg'><path fill='#78c346' d='M24 0c13.3 0 24 10.7 24 24S37.3 48 24 48 0 37.3 0 24C0 10.8 10.7 0 23.9 0h.1z'/><path fill='#fff' d='M20 36.4L6.7 23.1l3.6-3.6 9.7 9.9 17.7-17.9 3.6 3.6L20 36.4z'/></svg>"));
-    }
+  .header-container {
+    flex-shrink: 1;
+    margin-left: var(--terra-notification-dialog-header-container-margin-left, 0.5rem);
+  }
 
-    > .error {
-      background: var(--terra-notification-dialog-error-icon-background, inline-svg("<svg viewBox='0 0 49 49' xmlns='http://www.w3.org/2000/svg'><path fill='#e50000' d='M24 0c13.3 0 24 10.7 24 24S37.3 48 24 48 0 37.3 0 24C0 10.8 10.7 0 23.9 0h.1z'/><path fill='#fff' d='M7 21.5h34v5H7v-5z'/></svg>"));
-    }
+  .signal-word {
+    color: var(--terra-notification-dialog-signal-word-color, #1c1f21);
+    font-size: var(--terra-notification-dialog-signal-word-font-size, 1.42857rem);
+    font-weight: bold;
+    letter-spacing: -0.018rem;
+    line-height: 1.15;
+    min-height: var(--terra-notification-dialog-signal-word-min-height, 1.64285rem);
   }
 
   .title {
-    color: var(--terra-notification-dialog-title-color, #000);
-    flex: 0 0 auto;
-    flex-wrap: wrap;
-    font-size: var(--terra-notification-dialog-title-font-size, 1.07142rem);
-    font-weight: var(--terra-notification-dialog-title-font-weight, bold);
-    margin-top: var(--terra-notification-dialog-title-margin-top, 0.21428rem);
-    text-align: var(--terra-notification-dialog-title-text-align, left);
+    color: var(--terra-notification-dialog-title-color, #262626);
+    font-size: var(--terra-notification-dialog-title-font-size, 1.2857rem);
+    font-weight: var(--terra-notification-dialog-title-font-weight, normal);
+    letter-spacing: -0.018rem;
+    line-height: 1.16667;
+    margin: var(--terra-notification-dialog-title-margin, 0);
+    min-height: var(--terra-notification-dialog-title-min-height, 1.5rem);
     width: 100%;
     word-wrap: break-word;
   }
 
+  .body {
+    background-color: var(--terra-notification-dialog-body-background-color, #fff);
```

set to blank or transparent, so not to cover the floating header background
```suggestion
    background-color: var(--terra-notification-dialog-body-background-color, transparent);
```

---

updated: https://github.com/cerner/terra-framework/pull/1105/commits/c78f842fd0620606d38c4b19cf5cdfc8c76e50e0#diff-199a5a3214c6de6b926614643935d3a5R110